app/extensions/safe/test/e2e/safe.spec.js

```diff
@@ -104,6 +103,26 @@ describe( 'SAFE network webFetch operation', async () =>
         expect( parsedUrl.protocol ).toBe( 'safe:' );
     } );
 
+    it( 'validates URL before loading', async () =>
```

Comment:
This should be a `peruse` level test, @hunterlester. The `safe` portion of the test isn't so needed as just any invalid URL (as it goes off of protocols declared in the package.json). So I'd suggest we move it there (and amend the test to be less 'safe')